### PR TITLE
Modernised Uselessbot.. Also added gradle and config.json

### PR DESCRIPTION
**Pull request by MDWay**
Hey Sanduhr32!
Ist der Pull gut genug?

### CODE REVIEW

File: src/main/java/com/sanduhr/discord/Lib.java [hunk 1]
```diff
@@ -88,34 +79,33 @@
...
+        permMap.put("mention_@e", Permission.MESSAGE_MENTION_EVERYONE);
+        permMap.put("reactions", Permission.MESSAGE_ADD_REACTION);
+        permMap.put("read", Permission.MESSAGE_READ);
+        permMap.put("read_history", Permission.MESSAGE_HISTORY);
+        permMap.put("create_invites", Permission.CREATE_INSTANT_INVITE);
+        permMap.put("manage_channel", Permission.MANAGE_CHANNEL);
+        permMap.put("manage_perms", Permission.MANAGE_PERMISSIONS);
         permMap.put("manage_webhooks", Permission.MANAGE_WEBHOOKS);
-        permMap.put("manage_msgs",     Permission.MESSAGE_MANAGE);
+        permMap.put("manage_msgs", Permission.MESSAGE_MANAGE);
```
Comment (Sanduhr32) [inline src/main/java/com/sanduhr/discord/Lib.java]: Please undo destroying my format

File: src/main/java/com/sanduhr/discord/Useless.java [hunk 2]
```diff
@@ -4,71 +4,54 @@
...
-                int i = 0;
-                System.err.println("Invalid");
-                for (Character c : in.toCharArray()) {
-                    i += c.hashCode();
-                }
-                System.exit(i);
-        }
-        /**
-         * {@link net.dv8tion.jda.core.Permission.MESSAGE_MANAGE}
-         */
```
Comment (Sanduhr32) [inline src/main/java/com/sanduhr/discord/Useless.java]: Undo removing experimental startup check

File: src/main/java/com/sanduhr/discord/Useless.java [hunk 3]
```diff
@@ -4,71 +4,54 @@
...
+        }
     }
+
+
     public static void restart() throws Exception {
-        JDA.shutdown(false);
-        JDA = j.buildBlocking();
-    }
-    public static void reload() throws Exception {
-        restart();
```
Comment (Sanduhr32) [inline src/main/java/com/sanduhr/discord/Useless.java]: Add a method where you can shutdown JDA instances may use varargs

File: src/main/java/com/sanduhr/discord/utils/Channel/PrivateChannelUtils.java [hunk 4]
```diff
@@ -17,27 +19,30 @@
...
 
-        return users.stream().map(User::getPrivateChannel).collect(Collectors.toList());
+        return users.stream().map(user -> user.openPrivateChannel().complete()).collect(Collectors.toList());
     }
 
     /**
      * Returns a String of the Names from the private channels
+     *
      * @param privateChannels as List or Collection
      */
```
Comment (Sanduhr32) [inline src/main/java/com/sanduhr/discord/utils/Channel/PrivateChannelUtils.java]: Class is outdated will update after merging PR

File: src/main/java/com/sanduhr/discord/utils/Logutils.java [hunk 5]
```diff
@@ -112,15 +88,27 @@ public void onError(SimpleLog simpleLog, Throwable throwable) {
...
+    public static void setTextChannel(TextChannel TextChannel) {
+        if (TextChannel == null) {
+            try {
+                throw new NullPointerException("Hey! You spoon! NULL isn't a net.dv8tion.jda.core.entities.TextChannel!");
+            } catch (NullPointerException ex) {
+                ex.printStackTrace();
+            }
+        }
+        textChannel = TextChannel;
+    }
```
Comment (Sanduhr32) [inline src/main/java/com/sanduhr/discord/utils/Logutils.java]: Class is outdated will update after merging PR

**Comment by gerd2002** [comment]
~~I'd merge this, but Sanduhr requested changes.~~

**Comment by Sanduhr32** [comment]
I WOULD MERGE IT TOO @gerd2002 but i would like som chang

**Pull request merged**